5.1/5.1/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+class Program
+{
+    static void Main(string[] args)
+    {
+        string iraida = "пирамида";
+        string v = iraida.Substring(5, 1) + iraida.Substring(2, 2) + iraida.Substring(5, 1) + iraida.Substring(6,2);
+        Console.WriteLine(v);
+
+        string dima = "пирамида";
+        string V = dima.Substring(6, 1) + dima.Substring(5, 1) + dima.Substring(4, 1) + dima.Substring(3, 1);
+        Console.WriteLine(V);
```

Comment:
И снова неудачные именования. Лучше бы так
```suggestion
        string piramida = "пирамида";
        string iraida = piramida.Substring(5, 1) + piramida.Substring(2, 2) + piramida.Substring(5, 1) + piramida.Substring(6,2);
        Console.WriteLine(iraida);

        string dima = piramida.Substring(6, 1) + piramida.Substring(5, 1) + piramida.Substring(4, 1) + piramida.Substring(3, 1);
        Console.WriteLine(dima);
```